frontend/src/components/UploadStatus/UploadStatus.tsx

```diff
@@ -28,7 +32,44 @@ export function UploadStatus(): React.ReactElement {
             ? "Added"
             : ""}
         </span>
-        {uploadState.state === "parsed" && <SuccCheckmark />}
+        {(uploadState.state === "parsed" || uploadState.state === "added") && (
+          <div className="uploadStatus__statusIcons">
+            <div className="uploadStatus__statusIcon">
+              <SuccCheckmark />
+            </div>
+            <div
+              className="uploadStatus__resetIcon"
+              onClick={() => {
+                setExcelFile(undefined);
+                if (!isAdded(uploadState)) {
+                  notificationsUpdater({
+                    type: "Add",
+                    notificationType: "Error",
+                    element: manualRemove => (
+                      <div className="uploadingForm__resetNotify">
+                        <span>You have reset all completed fields</span>
+                        <Button
+                          type="text"
+                          className="uploadingForm__resetNotifyBtn"
+                          onClick={() => {
+                            setExcelFile(uploadState);
+                            manualRemove();
+                          }}
+                        >
+                          Undo
+                        </Button>
+                      </div>
+                    ),
+                  });
+                }
+              }}
+            >
+              <Tooltip text="Reset file">
```

Comment:
In design this tooltip says "Clear file field", did you change it intentionally? I don't have a preference here btw, I don't know what's better, just pointing out the difference. So I won't object if we keep "Reset file".